### PR TITLE
Closes #147 | implemented dataloader for gatitos dataset

### LINKED ISSUE #147: Create dataset loader for GATITOS
Dataloader name: `gatitos/gatitos.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?gatitos

|     Dataset| gatitos |
|-------------|---|
| Description | The GATITOS (Google's Additional Translations Into Tail-languages: Often Short) dataset is a high-quality, multi-way parallel dataset of tokens and short phrases. This dataset consists in 4,000 English segments (4,500 tokens) that have been translated into each of 173 languages, 170 of which are low-resource, 23 are spoken in Southeast Asia. This dataset contains primarily short segments: 93% single tokens, and only 23 sentences (0.6%) have over 5 tokens. As such it is best thought of as a multilingual lexicon, rather than a parallel training corpus. The source text is frequent words in the English Language, along with some common phrases and short sentences. Care has been taken to ensure that they include good coverage of numbers, months, days of the week, swadesh words, and names of the languages themselves (including the endonym). |
| Subsets | ace, ban, bbc, bew, bjn, bts, btx, bug, cnh, hil, iba, ilo, kac, lus, mad, mak, meo, min, pag, pam, shn, tet, war |
| Languages | ace, ban, bbc, bew, bjn, bts, btx, bug, cnh, hil, iba, ilo, kac, lus, mad, mak, meo, min, pag, pam, shn, tet, war |
| Tasks | Machine Translation |
| License | Creative Commons Attribution 4.0 (cc-by-4.0) |
| Homepage | https://github.com/google-research/url-nlp/blob/main/gatitos/README.md |
| HF URL | - |
| Paper URL | https://arxiv.org/abs/2303.15265 |

### PR DESCRIPTION
**Pull request by ssun32**
Closes #147 

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [x] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

**Comment by sabilmakbar** [comment]
Mind adding `__init__.py`?

**Comment by ssun32** [comment]
> Mind adding `__init__.py`?

@sabilmakbar  Done

File: seacrowd/sea_datasets/gatitos/gatitos.py
Comment (sabilmakbar) [inline seacrowd/sea_datasets/gatitos/gatitos.py:63]: can you rename it to `_URLS` so the variable convention is the same with the others?

File: seacrowd/sea_datasets/gatitos/gatitos.py
Comment (sabilmakbar) [inline seacrowd/sea_datasets/gatitos/gatitos.py:93]: since this is a MT task, we can remove this.
```suggestion
```
and for testing, you can add the `subset_id` using this:
```bash
python -m tests.test_seacrowd seacrowd/sea_datasets/gatitos/gatitos.py --subset_id "gatitos_$source_lang_$target_lang"
```

**Review by sabilmakbar — CHANGES_REQUESTED** [review]
few review notes

and after that, don't forget to format your code using this:
```bash
make check_file=seacrowd/sea_datasets/gatitos/gatitos.py
```

the implementation is running fine, aside from these reviews

File: seacrowd/sea_datasets/gatitos/gatitos.py
Comment (ssun32) [reply seacrowd/sea_datasets/gatitos/gatitos.py]: @sabilmakbar Thanks for your review. Pushed a new commit with your suggestions.

**Review by ljvmiranda921 — APPROVED** [review]
Dataloader works! I think some of the formatting comments were already pointed out by @sabilmakbar :) LGTM, let's just wait for @sabilmakbar 's approval.

**Review by sabilmakbar — APPROVED** [review]
All good, LGTM! Thanks @ssun32 for implementing this and @ljvmiranda921 for reviewing this. Merging it now...

**Pull request merged**
